File: Data Structures & Algorithms/is-anagram/submission-1.java
class Solution {
    public boolean isAnagram(String s, String t) {
        s = s.toCharArray.sort();
        t = t.sort();

        for (int i = 0 ; i < s.length() ; i++ ){
            if (!(s.charAt(i) == t.charAt(i))) {
                return false;
            }
        }
        return true;
    }
}

File: Data Structures & Algorithms/validate-parentheses/submission-1.py
class Solution:
    def isValid(self, s: str) -> bool:
        
        i = 0
        j = len(s)-1

        if len(s) % 2 !=0 :
            return False

        while i < j : 
            if s[i] == "{" and s[j] == "}":
                i+=1
                j-=1
            elif s[i] == "(" and s[j] == ")":
                i+=1
                j-=1
            elif s[i] == "[" and s[j] == "]":
                i+=1
                j-=1
            else :
                return False
        return True
                